terraform/ecs.tf

```diff
@@ -1,138 +1,9 @@
-# Define the cluster
```

Comment:
By using the yaml which is fully private and unavailable, we are missing to share/preserve the information that we used to have here and was ok to be public.